CHANGELOG-13.0.md

```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
 <!-- Add generated changelog below this line -->
+
+## [v13.0.0](https://github.com/shopsys/shopsys/compare/v12.0.0...v13.0.0) (2023-10-09)
+
+### :construction: Changes that require additional implementation if you are using Frontend API
+* Implemented SEO category functionality on storefront by @sebaholesz in https://github.com/shopsys/shopsys/pull/2654
+* improvements to Urql client and query error handling by @sebaholesz in https://github.com/shopsys/shopsys/pull/2659
+* [shopsys] graphql query is not in transaction by @stanoMilan in https://github.com/shopsys/shopsys/pull/2516
+
+### :sparkles: Enhancements and features
+* [shopsys] added various features by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2622
```

Comment:
I think like we should provide little more detail

Reply:
I have updated the texts after consultation with @pk16011990